test/integration/endpointslice/endpointslicemirroring_test.go

```diff
@@ -80,7 +80,7 @@ func TestEndpointSliceMirroring(t *testing.T) {
 	informers.Start(ctx.Done())
 	go epController.Run(ctx, 5)
 	go epsController.Run(5, ctx.Done())
-	go epsmController.Run(5, ctx.Done())
+	go epsmController.Run(ctx, 5)
```

Comment:
While at it, convert these tests to per-test output: create a `_, ctx := ktesting.NewTestContext(t)` for each test and use that context instead of `context.Background()`.